apps/codeforafrica/src/lib/data/seo.js

```diff
@@ -41,7 +41,7 @@ export default function getPageSeoFromMeta(page, settings) {
     title,
     description,
     type: "website",
-    site_name: siteTitle,
+    site_name: siteTitle ?? null,
```

Comment:
```suggestion
    site_name: siteTitle,
```

Since siteTitle will now never be undefined.